forge/db/models/Application.js

```diff
@@ -7,11 +7,6 @@ const { DataTypes } = require('sequelize')
 module.exports = {
     name: 'Application',
     schema: {
-        id: {
```

Comment:
Does an application not need a primary key?

Reply:
Sequelize will automatically add a field called `id` if there isn't a field already marked as a primary key (with basically the same constraints as the one removed)